rudder-core/src/main/scala/com/normation/rudder/services/policies/DataStructures.scala

```diff
@@ -378,6 +378,13 @@ final object Policy {
     path.replaceFirst(subPath, s"${p.technique.id.name.value}/${p.id.getRudderUniqueId}")
```

Comment:
Is their a case where we want to make a unique dest *without* replacing the tag? I think not (and it would avoid to walk the path several times)